src/main.py

```diff
@@ -2716,6 +2717,15 @@ def post_all(self, o):
         email = post["email"]
         signature = post["signature"]
         quicklinks = post["quicklinksid"]
+        rawfavouritereports = post["favouritereportsid"]
```

Comment:
minor thing, but the name of the feature was quickreports rather than favourite reports, even though they equate to the same thing. I would call this variable just quickreports